sbt-jib/src/main/scala/de/gccc/jib/JibPlugin.scala

```diff
@@ -158,130 +164,150 @@ object JibPlugin extends AutoPlugin {
+              jibImageFormat.value,
+              jibEnvironment.value,
+              jibLabels.value,
+              jibTags.value,
+              jibUser.value,
+              jibUseCurrentTimestamp.value
+            )
+          case None =>
+            streams.value.log.error("could not create jib tar image, cause path is not set")
+        }
+      },
+    jibJavaDockerBuild := PluginCompat.uncached {
+      implicit val conv: FileConverter = fileConverter.value
+      SbtJavaDockerBuild.task(
+        target.value,
+        streams.value.log,
+        Private.sbtConfiguration.value,
+        jibBaseImageCredentialHelper.value,
+        jibJvmFlags.value,
+        jibTcpPorts.value,
+        jibUdpPorts.value,
+        jibArgs.value,
+        jibEnvironment.value,
+        jibLabels.value,
+        jibTags.value,
+        jibUser.value,
+        jibUseCurrentTimestamp.value,
+        jibPlatforms.value
+      )
     },
+    jibJavaImageBuild := PluginCompat.uncached {
+      implicit val conv: FileConverter = fileConverter.value
+      SbtJavaImageBuild.task(
+        target.value,
+        streams.value.log,
+        Private.sbtConfiguration.value,
+        jibBaseImageCredentialHelper.value,
+        jibTargetImageCredentialHelper.value,
+        jibJvmFlags.value,
+        jibTcpPorts.value,
+        jibUdpPorts.value,
+        jibArgs.value,
+        jibImageFormat.value,
+        jibEnvironment.value,
+        jibLabels.value,
+        jibTags.value,
+        jibUser.value,
+        jibUseCurrentTimestamp.value,
+        jibPlatforms.value
+      )
+    },
+    jibJavaTarImageBuild :=
+      spaceDelimited("<path>").map { parsed =>
+        implicit val conv: FileConverter = fileConverter.value
+        parsed.headOption match {
+          case Some(pathString) =>
+            SbtJavaTarImageBuild.task(
+              target.value,
+              new File(pathString),
+              streams.value.log,
+              Private.sbtConfiguration.value,
+              jibBaseImageCredentialHelper.value,
+              jibJvmFlags.value,
+              jibTcpPorts.value,
+              jibUdpPorts.value,
+              jibArgs.value,
+              jibImageFormat.value,
+              jibEnvironment.value,
+              jibLabels.value,
+              jibTags.value,
+              jibUser.value,
+              jibUseCurrentTimestamp.value,
+              jibPlatforms.value
+            )
+          case None =>
+            streams.value.log.error("could not create jib java tar image, cause path is not set")
+        }
+      },
     jibDockerBuild     := jibDockerBuild.dependsOn(Compile / compile).value,
     jibImageBuild      := jibImageBuild.dependsOn(Compile / compile).value,
     jibJavaDockerBuild := jibJavaDockerBuild.dependsOn(Compile / compile).value,
     jibJavaImageBuild  := jibJavaImageBuild.dependsOn(Compile / compile).value
   )
 
+  private implicit val imageReferenceFormat: JsonFormat[ImageReference] =
```

Comment:
Maybe we could make these tasks return the Jib `ImageReference#toString()` and avoid all this?